server/app/controllers/search.js

```diff
@@ -0,0 +1,27 @@
+var Search = require('mongoose').model('Place');
```

Comment:
You shouldn't create separate controller for search, use `controller/places.js` instead.